api/resources_portal/models/attachment.py

```diff
@@ -0,0 +1,37 @@
+from django.db import models
+
+
+class Attachment(models.Model):
+    class Meta:
+        db_table = "attachments"
+        get_latest_by = "created_at"
+
+    objects = models.Manager()
+
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    filename = models.TextField(blank=False, null=True, help_text="The name of the attachment.")
+
+    s3_bucket = models.CharField(max_length=255, blank=True, null=True)
+    s3_key = models.CharField(max_length=255, blank=True, null=True)
+
+    sequence_map_for = models.ForeignKey(
+        "resources_portal.Material",
+        blank=False,
+        null=True,
+        on_delete=models.SET_NULL,
+        related_name="sequence_maps",
+        help_text="The cell line this seq_map is for. Only valid for seq_map attachments.",
```

Comment:
Should attachment have another field called "type" or something similar which specifies the kind of attachment it is? Might make it easier to process on the front-end.

Reply:
I think we will want to add description instead. We should generally try to avoid getting attachments directly and only be getting them from some sort of association to what they are attached to.